src/components/document/student-group-view.tsx

```diff
@@ -38,66 +40,76 @@ interface IProps {
   setGroupId: (groupId: string) => void;
 }
 export const StudentGroupView:React.FC<IProps> = ({ groupId, setGroupId }) => {
-  const user = useUserStore();
-  const groups = useGroupsStore();
+  const {user, groups, documents} = useStores();
   const [focusedGroupUser, setFocusedGroupUser] = useState<GroupUserModelType | undefined>();
-  const [groupViewContext, setGroupViewContext] = useState<string | null>(null);
   const selectedGroupId = groupId || (groups.allGroups.length ? groups.allGroups[0].id : "");
+  const groupUsers = getGroupUsers(user.id, groups, documents, selectedGroupId);
+  const group = groups.getGroupById(selectedGroupId);
 
   const handleSelectGroup = (id: string) => {
     Logger.log(LogEventName.VIEW_GROUP, {group: id, via: "group-document-titlebar"});
     setGroupId(id);
     setFocusedGroupUser(undefined);
-    setGroupViewContext(null);
-  };
-  const handleFocusedGroupUserChange = (selectedGroupUser: GroupUserModelType | undefined) => {
-    setFocusedGroupUser(selectedGroupUser);
-  };
-
-  const handleToggleContext = (context: string | null, selectedGroupUser: GroupUserModelType | undefined) => {
-    setGroupViewContext(context);
-    setFocusedGroupUser(selectedGroupUser);
   };
 
   const GroupViewTitlebar: React.FC<IGroupViewTitlebarProps> = ({ selectedId, onSelectGroup }) => {
     return (
       <div className={`titlebar student-group group`}>
         <div className="actions">
-          { groups.allGroups
-              .filter(group => group.users.length > 0)
-              .map(group => {
-                return <GroupButton displayId={group.displayId} id={group.id} key={group.id}
-                                    selected={group.id === selectedId}
-                                    onSelectGroup={onSelectGroup} />;
+        {focusedGroupUser && group
+          ? <>
+              <GroupButton displayId={group.displayId} id={group.id} key={group.id}
+                            selected={group.id === selectedId}
+              />
+              { groupUsers.map(u => {
+                  const className = classNames("member-button", "in-student-group-view", `${u.context}`);
```

Comment:
```suggestion
                  const className = classNames("member-button", "in-student-group-view", u.context);
```
I also feel like the indentation in this section might be a little wonky but it's tough for me to tell.